### DISCUSSION TITLE
Pre-trained ViT-MaskRCNN on COCO detection dataset

### DISCUSSION THREAD

**Discussion by sfarkya** [facebookresearch/detectron2#4243] [Q&A]
Hello Dear Authors, 

Thank you for the great work! 
I have been reading these two papers on Self-supervised learning, 

1. Benchmarking Detection Transfer Learning with Vision Transformers (https://arxiv.org/pdf/2111.11429.pdf)
2. Masked Autoencoders Are Scalable Vision Learners (https://arxiv.org/pdf/2111.06377.pdf)
 and found ViT trained on COCO to be interesting in my research. 

Both the papers provide GitHub links  
1. https://github.com/facebookresearch/detectron2
2. https://github.com/facebookresearch/mae 

But it's mentioned in the paper that the code and model for ViT-MaskRCNN will be released but I am unable to track it down, I check the documentation of Detectron2 but no luck!

Could be please help me in locating the model and the training code or at least tell me if it's released or not? 

Best, 
Saurabh
